scipy/stats/stats.py

```diff
@@ -5436,7 +5441,10 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
     statistic: float
         The statistic calculated by the specified method:
```

Comment:
I think the sentence "The statistic calculated by the specified method" is enough, method should be described above

Reply:
Yes, remove altogether, since it is redundant and i think it is better to describe it under method above. Do you agree?

Reply:
I thought so at first but now I'm not sure. 

 I think we should have distributional information in the docstring somewhere. This seems like a good spot to document that-even though I haven't for the added methods I will if we agree this is the spot.

If we don't want to have the distributional information here, I'm fine with that too, but with multiple methods and varying distributions/laws we should document for someone using the test statistic output from this method as input somewhere else. 

What do you think @chrisb83 ?

Reply:
I would still argue that the output is clear when the user reads the description of the keyword "method", so repeating it here adds unnecessary / redundant information